tools/Azure.Mcp.Tools.Redis/src/Services/RedisService.cs

```diff
@@ -255,4 +349,60 @@ [.. resource.PrivateEndpointConnections.Select(connection => connection.Id.ToStr
 
         return resources;
     }
+
+    private static string GetCreateResourceBicepTemplate()
+    {
+        return """
+            {
+              "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+              "contentVersion": "1.0.0.0",
+              "parameters": {
+                "resourceName": {
+                  "type": "string"
+                },
+                "location": {
+                  "type": "string"
+                },
+                "skuName": {
+                  "type": "string"
+                },
+                "modules": {
+                  "type": "array",
+                  "defaultValue": []
+                }
+              },
+              "resources": [
+                {
+                  "type": "Microsoft.Cache/redisEnterprise",
+                  "apiVersion": "2025-05-01-preview",
+                  "name": "[parameters('resourceName')]",
+                  "location": "[parameters('location')]",
+                  "sku": {
+                    "name": "[parameters('skuName')]"
+                  },
+                  "properties": {
+                    "highAvailability": "Enabled",
+                    "minimumTlsVersion": "1.2"
+                  }
+                },
+                {
+                  "type": "Microsoft.Cache/redisEnterprise/databases",
+                  "apiVersion": "2025-05-01-preview",
+                  "name": "[format('{0}/default', parameters('resourceName'))]",
+                  "properties": {
+                    "clientProtocol": "Encrypted",
+                    "clusteringPolicy": "OSSCluster",
+                    "evictionPolicy": "VolatileLRU",
+                    "deferUpgrade": "NotDeferred",
+                    "accessKeysAuthentication": "Disabled",
```

Comment:
Don't we need to tokenize properties like these to honor the settings passed in?